Ports/SDL2_compat/package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SDL2_compat
```

Comment:
Upstream named this piece of software `sdl12-compat`, and I don't really see a reason to deviate from that.

Reply:
I did it for consistency, but I can change that. Fixing.